client/platform/desktop/background.ts

```diff
@@ -61,6 +62,7 @@ async function createWindow() {
     console.error(`Server listening on ${address}:${port}`);
   });
   ipcListen();
+  initializeRemote();
```

Comment:
What does this added line do?

Reply:
https://github.com/electron/remote -- remote package was removed from electron core.